addons/web/static/src/core/l10n/translation.js

```diff
@@ -17,9 +19,13 @@ export const translationIsReady = new Deferred();
  * @param {string} term
  * @returns {string}
  */
-export function _t(term) {
+export function _t(term, ...values) {
     if (translatedTerms[translationLoaded]) {
-        return translatedTerms[term] || term;
+        const translation = translatedTerms[term] ?? term;
+        if (values.length === 0) {
+            return translation;
+        }
+        return sprintf(translation, ...values);
     } else {
         return new LazyTranslatedString(term);
```

Comment:
You're not forwarding the values correctly here

Reply:
whoops, fixed 🤠